modules/error_handling/include/hephaestus/error_handling/panic.h

```diff
@@ -66,15 +81,22 @@ class Panic : public std::runtime_error {
 /// @param message A message describing the error and what caused it
 /// @param location Location in the source where the error was triggered at
 template <typename... Args>
-void panic(internal::StringLiteralWithLocation<Args...> message, Args&&... args) {
-  std::string formatted_message = fmt::format(message.value, std::forward<Args>(args)...);
-#ifndef DISABLE_EXCEPTIONS
-  throw Panic{ std::move(formatted_message), message.location };
-#else
-  auto e = Panic{ std::move(formatted_message), message.location };
-  CHECK(false) << fmt::format("[ERROR {}] at {}:{}", e.what(), message.location.file_name(),
-                              message.location.line());
-#endif
+void panic(error_handling::detail::StringLiteralWithLocation<Args...> message, Args&&... args) {
+  auto formatted_message = fmt::format(message.value, std::forward<Args>(args)...);
+  auto location = std::string(utils::string::truncate(message.location.file_name(), "modules")) + ":" +
+                  std::to_string(message.location.line());
+
+  // We want to make sure the log from panic is the last one before termination so we flush before and after
+  telemetry::flushLogEntries();
```

Comment:
is that really needed? the logging should happen in the correct order no independant if it is flushed or not.